generate-clients.sh

```diff
@@ -48,10 +49,14 @@ function generate_client() {
       --git-host "github.com" --git-user-id RedHatInsights --git-repo-id patchman-clients \
       --type-mappings DateTime=string \
       -o /local/$NAME
+
+    if [ "$NAME" == "inventory" ] ; then
+        sed -i 's/uNKNOWNBASETYPE/createCheckIn/g; s/UNKNOWN_BASE_TYPE/CreateCheckIn/g;' $NAME/api_hosts.go
+    fi
 }
 
 
-$CONTAINER_TOOL pull $CONTAINER
+$CONTAINER_TOOL image exists $CONTAINER || $CONTAINER_TOOL pull $CONTAINER
```

Comment:
Can not we just remove this line? I think image is pulled automatically when it's not found among local images.

Reply:
docker/podman pull was added to force image update before creating container.
But since we actually pinned the stable version instead of using latest it might be redundant.